packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.spec.js

```diff
@@ -0,0 +1,271 @@
+import CollaboratorsEditOptions from '@files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue'
+import { createLocalVue, mount, shallowMount } from '@vue/test-utils'
+import GetTextPlugin from 'vue-gettext'
+import Vuex from 'vuex'
+import DesignSystem from 'owncloud-design-system'
+import VueSelect from 'vue-select'
+import RoleItem from '@files/src/components/SideBar/Shared/RoleItem.vue'
+import { DateTime } from 'luxon'
+
+import { roles } from '@files/tests/__fixtures__/collaborators.js'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
+localVue.use(Vuex)
+localVue.use(VueSelect)
+localVue.use(RoleItem)
+localVue.use(GetTextPlugin, {
+  translations: 'does-not-matter.json',
+  silent: true
+})
+
+const selectors = {
+  roleLabel: 'label',
+  roleSelect: '.files-collaborators-role-button-wrapper',
+  expirationDate: '#files-collaborators-collaborator-expiration-input'
+}
+
+describe('Collaborator Edit Options', () => {
+  describe('Role Selection', () => {
+    let wrapper = getMountedWrapper({
+      user: 'user0'
+    })
+
+    it('should show the label for the role', () => {
+      const roleLabel = wrapper.find(selectors.roleLabel)
+
+      expect(roleLabel.exists()).toBeTruthy()
+      expect(roleLabel.attributes()).toMatchObject({
+        class: 'oc-label',
+        for: 'files-collaborators-role-button'
+      })
+
+      expect(roleLabel.text()).toBe('Role')
```

Comment:
use snapshot tests to assert text
```suggestion
      expect(roleLabel).toMatchSnapshot()
```